misc/python/materialize/buildkite_insights/step_durations/build_step.py

```diff
@@ -85,6 +87,7 @@ def _extract_build_step_data_from_build(
             job_passed,
             exit_status=exit_status,
             retry_count=retry_count,
+            web_url=f"{build_data['web_url']}#{job['id']}",
```

Comment:
Could you please use this field in `misc/python/materialize/buildkite_insights/step_durations/analysis.py` in line 127?